apps/condo/domains/common/utils/tables.utils.ts

```diff
@@ -380,3 +382,12 @@ export const convertColumns = (
 export const getSorterMap: SorterMapType = (sorters) => {
     return Object.assign({}, ...sorters.map((sorter) => ({ [sorter.columnKey]: sorter.order })))
 }
+
+export const sorterUnitName: (a: Ticket, b: Ticket) => number = (a: Ticket, b: Ticket) => {
+    const firstUnitName = get(a, 'unitName', null)
+    const secondUnitName = get(b, 'unitName', null)
+    if (!firstUnitName && !secondUnitName) return 0
+    if (!secondUnitName) return -1
+    if (!firstUnitName) return 1
+    return Number(firstUnitName) - Number(secondUnitName)
```

Comment:
there could be case with NaN, I think you should handle it too. And please add tests for this sorter function.

Reply:
first 3 conditions also check NaN